packages/legacy/core/App/components/misc/CredentialCard11.tsx

```diff
@@ -240,6 +233,9 @@ const CredentialCard11: React.FC<CredentialCard11Props> = ({
   const backgroundColorIfErrorState = (backgroundColor?: string) =>
     error || predicateError || isProofRevoked ? ColorPallet.notification.errorBorder : backgroundColor
 
+  const backgroundColorIfRevoked = (backgroundColor?: string) =>
+    isProofRevoked ? ColorPallet.notification.errorBorder : backgroundColor
+
```

Comment:
Should we get rid of `backgroundColorIfErrorState` entirely if it's only being used in that `fontColorWithHighContrast` function and just use the new `backgroundColorIfRevoked`? Have you checked that it looks right when a credential is just missing entirely?